src/test/scala/com/databricks/spark/csv/CsvSuite.scala

```diff
@@ -852,6 +853,38 @@ abstract class AbstractCsvSuite extends FunSuite with BeforeAndAfterAll {
     assert(results.toSeq.map(_.toSeq) === expected)
   }
 
+  test("DSL allows for setting maxColsPerChar and expect error") {
+    val parser = new CsvParser()
+      .withDelimiter(',')
+      .withUseHeader(true)
+      .withParseMode(ParseModes.FAIL_FAST_MODE)
+      .withMaxCharsPerCol(5000)
```

Comment:
If this option is only for `univocity`, it would be better if we test this only when `parseLib` is `univocity`, which can be like this:
```scala
test("DSL allows for setting maxColsPerChar and expect error") {
  assume(parserLib == "UNIVOCITY", "maxCharsPerCol option is only " +
    "for univocity parser.")
  val parser = new CsvParser()
...
```